### PR TITLE
Adding a few unittests for DBN, JPD and UndirectedGraph

### PR DESCRIPTION
**Pull request by liquidmetal**

### CODE REVIEW

**Comment by liquidmetal** [comment]
Landscape.io seems stuck - any idea why?

**Comment by yashu-seth** [comment]
There seems to be a bug in landscape. I don't think we can do much about it.

**Comment by liquidmetal** [comment]
It seems it has been stuck since Jan 13. Did anyone file an issue at https://github.com/landscapeio/landscape-issues/issues/new ? Should we turn off Landscape.io until things become stable?

**Comment by yashu-seth** [comment]
Lets see what @ankurankan feels about it.

File: pgmpy/tests/test_factors/test_JointProbabilityDistribution.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            JointProbabilityDistribution(['a'], [2], [[0.4], [0.4]])
+
+        with self.assertRaises(ValueError):
+            JointProbabilityDistribution(['a', 'b'], [2, 2], [[0.4, 0.4], [0.6, 0.6]])
+
+        jpd = JointProbabilityDistribution(['a'], [2], [[0.4], [0.6]])
+        self.assertEqual(jpd.variables, ['a'])
+        np_test.assert_equal(jpd.values, np.array([0.4, 0.6]))
+
+
```
Comment (khalibartan) [inline pgmpy/tests/test_factors/test_JointProbabilityDistribution.py:21]: @liquidmetal  we have got tests for JointProbabilityDistribution Class https://github.com/pgmpy/pgmpy/blob/dev/pgmpy/tests/test_factors/test_Factor.py#L448

**Comment by khalibartan** [comment]
@liquidmetal Update your dev branch, it is behind our dev. You will see then we have got tests for JointProbabilityDistribution . If you want to add test then you can add more tests there.

**Comment by liquidmetal** [comment]
My bad - I'll fix this and create a new PR.

**Pull request closed without merging**